scripts/release_issue_status/update_issue_body.py

```diff
@@ -0,0 +1,89 @@
+import re
+
+def update_issue_body(sdk_repo, rest_repo, issue_link):
+    # Get Issue Number
+    issue_number = int(issue_link.replace("https://github.com/Azure/sdk-release-request/issues/", ""))
+    issue_info = sdk_repo.get_issue(number=issue_number)
+    issue_body = issue_info.body
+
+    issue_body_list = [i for i in issue_body.split("\n") if i]
+    # Get the link in issue body
+    for row in issue_body_list:
+        if 'link' in row.lower():
+            link = row.split(":", 1)[-1].strip()
+            break
+
+    if link.count('https') > 1:
+        link = link.split(']')[0]
+        link = link.replace('[', "").replace(']', "").replace('(', "").replace(')', "")
+
+    try:
+        package_name, readme_link = get_pkname_and_readme_link(rest_repo, link)
```

Comment:
It seems that `get_pkname_and_readme_link` would not raise exception?

Reply:
I'm worried about the occurrence of new types of links and throwing exceptions, so I use catch exceptions